datasets/us/fda_restricted/us_fda_restricted.yaml

```diff
@@ -0,0 +1,44 @@
+title: US Restricted Clinical Investigators
+entry_point: crawler.py
+prefix: us-fdares
+coverage:
+  frequency: daily
+  start: "2024-12-10"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Clinical researchers who have been prohibited from participating in FDA-regulated clinical investigations.
+description: |
+  The FDA regulates clinical investigators who conduct studies intended to provide evidence on the safety
+  and effectiveness of investigational medical products. Clinical investigators must follow applicable
+  statutes and regulations to ensure the integrity of clinical data and, for studies involving human subjects,
+  to protect their rights, safety, and welfare. If the FDA determines that a clinical investigator has violated
+  these requirements—such as through data falsification or noncompliance—it may initiate a disqualification
```

Comment:
When we switch from context to what this contains, it's a nice opportunity for a paragraph break

```suggestion
  to protect their rights, safety, and welfare.
  
  If the FDA determines that a clinical investigator has violated
  these requirements—such as through data falsification or noncompliance—it may initiate a disqualification
```